plasma_framework/python_tests/tests/tests_utils/plasma_framework.py

```diff
@@ -202,8 +202,31 @@ def challengeStandardExit(self,
     def startFeeExit(self, token, amount):
         raise NotImplementedError
 
-    def startInFlightExit(self, in_flight_tx, input_txs, input_txs_inclusion_proofs, in_flight_tx_sigs):
-        raise NotImplementedError
+    def startInFlightExit(self,
+                          in_flight_tx,
+                          input_txs,
+                          input_txs_inclusion_proofs,
+                          in_flight_tx_sigs,
+                          input_utxos_pos,
+                          **kwargs):
+
+        in_flight_tx_witnesses = in_flight_tx_sigs
+
+        # The following arguments are to be used in the future
```

Comment:
We are removing these args actually. I think should be seen while rebase to master.